lib/ext/heroku/command/addons.rb

```diff
@@ -225,6 +225,14 @@ def add
       end
     end
 
+    # addons:add ADDON
+    #
+    # WARNING: `heroku addons:add has been deprecated. Please use `heroku addons:create` instead.
```

Comment:
I can't recall if this works, but if we leave off the command description does it cease showing in the help text? It might be good to not show it in the help text if possible. @geemus is that how it works?